### PR TITLE
Update discovering-spicedb.mdx

### PR DESCRIPTION
**Pull request by KambangSinclaire**
Fixed some typing errors

### CODE REVIEW

**Comment by vroldanbet** [comment]
@KambangSinclaire we appreciate the change 🙏🏻  I'm afraid `authzed/docs` are now deprecated - `authzed.com/docs` is now build from a internal repository.

**Comment by vroldanbet** [comment]
My bad, it appears this is still used as the source of truth for the docs

File: pages/spicedb/getting-started/discovering-spicedb.mdx [hunk 1]
```diff
@@ -39,12 +39,12 @@ For folks interested in a managed SpiceDB services and enterprise functionality,
 ### A brief SpiceDB history lesson
 
 In August 2020, the founders of AuthZed left [Red Hat], who had acquired their previous company [CoreOS].
-In the following month, they would write the first API-complete implementation of Zanzibar; project Arrakis was written in lazily-evaluated, type-annotated Python.
+In the following month, they would write the first API-complete implementation of Zanzibar; project Arrakis was written in lazily evaluated, type-annotated Python.
```
Comment (samkim) [inline pages/spicedb/getting-started/discovering-spicedb.mdx:42]: Please revert, "lazily-evaluated" is intended to be an adjective for Python

File: pages/spicedb/getting-started/discovering-spicedb.mdx [hunk 2]
```diff
@@ -57,8 +57,8 @@ You can read also read the [history of Google's Zanzibar project][zanzibar-histo
 Features that distinguish SpiceDB from other systems include:
 
 - Expressive [gRPC] and [HTTP/JSON] APIs for checking permissions, listing access, and powering devtools
-- A distributed, parallel graph-engine faithful to the architecture described in [Google's Zanzibar paper]
-- A flexible consistency model configurable [per-request] that includes resistance to the [New Enemy Problem]
+- A distributed, parallel graph engine faithful to the architecture described in [Google's Zanzibar paper]
+- A flexible consistency model configurable [per request] that includes resistance to the [New Enemy Problem]
```
Comment (samkim) [inline pages/spicedb/getting-started/discovering-spicedb.mdx:61]: Update the associated link definition below on line 69

**Review by samkim — CHANGES_REQUESTED** [review]

**Comment by vroldanbet** [comment]
Closing as stale

**Pull request closed without merging**